Sources/SwifterSwift/SwiftStdlib/FloatingPointExtensions.swift

```diff
@@ -81,3 +81,15 @@ public prefix func ± <T: FloatingPoint> (number: T) -> (T, T) {
     return 0 ± number
 }
 // swiftlint:enable identifier_name
+
+// swiftlint:disable identifier_name
+prefix operator √
+/// SwifterSwift: Square root of float.
+///
+/// - Parameter float: float value to find square root for
+/// - Returns: square root of given float.
+public prefix func √<T> (float: T) -> T where T: FloatingPoint {
```

Comment:
I wonder if its time to think about inline such trivial numeric operators like this, the implementation is trivial so maybe we should throw an `@inlinable` here (and possibly audit other places where we could do it as well). This is good especially for generic functions like this one because it would allow the compiler to specialize such functions on clients that install this library as a module.